compiler/packages/babel-plugin-react-compiler/src/HIR/HIR.ts

```diff
@@ -1445,6 +1445,7 @@ export enum ValueKind {
   Primitive = 'primitive',
   Global = 'global',
   Mutable = 'mutable',
+  ShallowMutable = 'shallowmutable',
```

Comment:
you'll need to update `mergeValueKinds()` in InferMutationAliasingEffects to account for the new type